tests/config/format-test.js

```diff
@@ -424,10 +424,16 @@ const insertCursor = (text, cursorOffset) =>
       text.slice(cursorOffset)
     : text;
 function format(originalText, originalOptions) {
-  const { text: input, options } = replacePlaceholders(
+  let { text: input, options } = replacePlaceholders(
     originalText,
     originalOptions
   );
+
+  options = {
+    pluginSearchDirs: false,
+    ...options,
+  };
```

Comment:
Can we keep using `const` instead of `let` and add property to `options`? I have concern to copy large object by spread syntax.
```js
options.pluginSearchDirs = false;
```

Reply:
https://github.com/prettier/prettier/pull/12322/commits/1c099928c84a51f3a8637a1ffe6e1a742d70b2d2

Reply:
Found a better place to put it.. https://github.com/prettier/prettier/pull/12322/commits/747ee772d96093347944bc40ae22f58f9a15a15b